### PR TITLE
User onboarding page

### PR DESCRIPTION
**Pull request by Manpreet-hub**
Added onboarding page and User-info page.
Added components with dummy data.

Screenshot (131)

Screenshot (132)

### CODE REVIEW

File: src/pages/UserInfo.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export const UserInfo = () => {
+  const time = new Date();
+  const hour = time.getHours();
+  const minute = time.getMinutes();
+  const minutes = minute / 10 < 1 ? `0${minute}` : minute;
+  const timeInfo = `${hour}: ${minutes}`;
+  const greetings = () => {
+    if (hour < 4) return " Night";
+    else if (hour < 12) return " Morning";
+    else if (hour < 16) return " Afternoon";
```
Comment (ashishnnnnn) [inline src/pages/UserInfo.js:12]: Liked this logic Implementation

**Review by ashishnnnnn — APPROVED** [review]
Code looks Clean..
Can be merged ..

File: src/components/Todo.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const Todo = () => {
+  return (
+    <>
+      <button class="btn-default btn-dark">Todo</button>
```
Comment (sharathnair9999) [inline src/components/Todo.js:4]: if it is a single element, u need not wrap it in empty fragments.

File: src/components/Weather.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const Weather = () => {
+  return (
+    <>
```
Comment (sharathnair9999) [inline src/components/Weather.js:3]: here as well. if the child inside the fragment is one parent element, you can omit the empty fragments.

**Review by sharathnair9999 — APPROVED** [review]
Everything looks good. Suggested trivial changes

File: src/components/Weather.js (hunk 3 above)
Comment (Manpreet-hub) [reply src/components/Weather.js:3]: yes, good point! , but this is just dummy data in the component, will work on it & remove empty fragments in the upcoming PRs.

**Pull request merged**